pybind11/__init__.py

```diff
@@ -10,9 +10,17 @@ def get_include(user=False):
     virtualenv = hasattr(sys, 'real_prefix') or \
         sys.prefix != getattr(sys, "base_prefix", sys.prefix)
 
+    # Are we running in a conda environment?
+    conda = os.path.exists(os.path.join(sys.prefix, 'conda-meta'))
+
     if virtualenv:
```

Comment:
@sdebionne maybe that's a stupid question but let me ask it anyway:
can one be inside conda and as well, due to it's pip support, within a virtualenv? If so, how to handle this gracefully?